internal/tasks/fleet_rollout.go

```diff
@@ -378,14 +378,6 @@ func (f FleetRolloutsLogic) replaceGitConfigParameters(device *api.Device, confi
 		errs = append(errs, fmt.Errorf("failed replacing parameters in path in git config %s: %w", gitSpec.Name, err))
 	}
 
-	if gitSpec.GitRef.MountPath != nil {
```

Comment:
1. The code for replacing the parameters was removed, but the [documentation](https://raw.githubusercontent.com/flightctl/flightctl/refs/heads/EDM-1188/docs/user/managing-devices.md) still refers to `mountPath`.
2. Some [example](https://github.com/flightctl/flightctl/blob/EDM-1188/examples/fleet-b.yaml#L22) still refer to it as well. 

The files I linked are just examples, please check if there are more places that refer to `mountPath`.

Reply:
This would be a breaking change, but it is unclear to me why we need to change the API to fix the bug. @noga-magen can you elaborate? Or even better, can we bring this up in our next group call?